reporting/api/src/main/java/org/jboss/windup/reporting/model/JarDependenciesReportModel.java

```diff
@@ -0,0 +1,33 @@
+package org.jboss.windup.reporting.model;
+
+import org.jboss.windup.graph.model.ProjectModel;
+
+import com.tinkerpop.blueprints.Direction;
+import com.tinkerpop.frames.Incidence;
+import com.tinkerpop.frames.modules.typedgraph.TypeValue;
+
+/**
+ * This represents Jar Dependencies report model
+ * 
+ * @author mnovotny
+ *
+ */
+@TypeValue(JarDependenciesReportModel.TYPE)
+public interface JarDependenciesReportModel extends ApplicationReportModel
+{
+    String TYPE = "jarDependenciesReport";
+
+    String JAR_REPORT_TO_PROJECT_MODEL = "jarDepsToProjectModel"; 
+
+    /**
+     * Contains all {@link ProjectModel}s that contain this file
+     */
+    @Incidence(label = JAR_REPORT_TO_PROJECT_MODEL, direction = Direction.OUT)
```

Comment:
Hmm, so this is for first time I create a such thing, so be patient with me ;) as I need to get it correctly.

I understand it that we need to create new vertex, call it `DependencyModel` with sha1sum property and some properties to identify archive (model or just name) and create a java handler method to get all duplicate archives. Is that correct?
What I am not sure is about adding an edge, which is in this case `JarDependencyReportToProjectEdgeModel` which I will rename to more suitable name (for instance `JarDependencyReportToDependencyEdgeModel`) as I will replace projectModel there to point to a `ArchiveModel`.